src/compiler/utilitiesPublic.ts

```diff
@@ -2005,9 +1998,6 @@ export function isLiteralTypeLiteral(node: Node): node is NullLiteral | BooleanL
 
 /**
  * Determines whether a node is an expression based only on its kind.
- * Use `isExpressionNode` if not in transforms.
- *
- * @internal
  */
 export function isExpression(node: Node): node is Expression {
     return isExpressionKind(skipPartiallyEmittedExpressions(node).kind);

```

Comment:
Unlike `isLiteralTypeLiteral` above, the `skipPartiallyEmittedExpressions(node)` here is fine, since a `PartiallyEmittedExpression` is still an `Expression`.

Reply:
Couldn't this cause issues if the `PartiallyEmittedExpression` node isn't a child of an `Expression` node? Or would that case never happen in the real world?

Reply:
I don't know the answer off the top of my head, but these nodes appear to only be created in transformed code (coming from the ts and es2015 transforms for downleveling), so if you're thinking about linting, I don't think you can actually encounter these nodes.